packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.tsx

```diff
@@ -107,6 +107,7 @@ export class Suggestions<T> extends BaseComponent<ISuggestionsProps<T>, ISuggest
       resultsMaximumNumber,
       resultsFooterFull,
       resultsFooter,
+      isResultsFooterVisible,
```

Comment:
move this to the bottom and set it to true. It will look like
```
let {
<other props>
suggestionsHeaderText,
isResultsFooterVisible = true
} = this.props;
```